fluentd/configs.d/filter-pre-mux-client.conf

```diff
@@ -5,6 +5,8 @@
   shared_key "#{File.open('/etc/fluent/muxkeys/shared_key') do |f| f.readline end.rstrip}"
   secure yes
   ca_cert_path /etc/fluent/muxkeys/ca
+  buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT']}"
```

Comment:
@nhosoi, is it really a concern to think a buffer chunk size of even 16 MB would take longer than 1 second to transmit?  Can't a 1 Gb/s link can transfer 125 MB in one second?  It does not seem to be the case that a low `flush_interval` will cause some kind of problem with an inability to transmit a chunk in that window.

If we are going to stick with the `memory` buffer, which is arguably a bad idea, we'll want to probably use no more than 512 MB of memory for a fluentd pod so that that pod is not obtrusive in its memory consumption.  With a low `flush_interval`, it is possible that chunks are queued to be flushed before they are filled.  So then the `buffer_queue_limit` and `buffer_chunk_limit` serve to form the upper-bound of memory usage.

If a `buffer_queue_limit` is reached, then an exception is raised, and the `json-file` reader or `journald` reader should stop reading more data until it can place data into the buffer queue.  Let's assert that that is the behavior we want going forward.